Test/Platform/Tests/CLR/mscorlib/systemlib/systemlib2/Utf8EncodingTests.cs

```diff
@@ -76,5 +76,24 @@ public MFTestResults Utf8EncodingTests_Test2()
 
             return result ? MFTestResults.Pass : MFTestResults.Fail;
         }
+
+        [TestMethod]
+        public MFTestResults Utf8EncodingTests_Test3()
+        {
+            // This tests involves a string with a special character
+
+            bool result = true;
+
+            string str = "AB\u010DAB";
+
+            byte[] data = new byte[4];
+            int count = UTF8Encoding.UTF8.GetBytes(str, 1, 3, data, 0);
+
+            result &= count == 4;
+
+            result &= (new string(UTF8Encoding.UTF8.GetChars(data)) == "B\u010DA");

```

Comment:
I agree with your point; however, all the other tests are implemented in this manner. When we start implementing new test suits we should do it right, but for now I think it is better to keep it uniform as we do not have any plan to fix the others.